decentraland/components/presentational/__snapshots__/MyAccountProgress.test.js.snap

```diff
@@ -0,0 +1,49 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MyAccountProgress renders the component 1`] = `
+<Component>
+  <ProgressBar
+    animated={true}
+    animationConfig={
+      Object {
+        "bounciness": 0,
```

Comment:
This seems like an interesting setting to play with

Reply:
Well yeah, but for some reason, when I start generating account, I cannot go back to see updates in real time, seems some bug with OnBoardingScreen I guess :/

Reply:
Ah ok good to know - thanks. cc @marcelomorgado

Reply:
@marcelomorgado Think we should extract this into an issue? Or is it related to a pre-existing one?